src/components/RecentJobs/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react';
+import JobCard from '../JobCard';
+import { JobContext } from '../../context';
+import SkillsItem from '../SkillsItem';
+
+function RecentJobs() {
```

Comment:
This section needs some work on mobile: https://dev.io-academy.uk/resources/jobsearch/home-mobile.png

Reply:
done

Reply:
Ok this is now wayyyy better. One final thing, can you hide the images on mobile? I think then it will be spot on.